Manual_for_achitecture_browser/index.asciidoc

```diff
@@ -0,0 +1,62 @@
+= Manual for architecture browser
```

Comment:
You should not use level-one headlines because when converting to HTML they become the page title and not the first headline. -> you have to increase the level of all headlines by 1